recipes/lyra/all/conanfile.py

```diff
@@ -25,15 +24,13 @@ def package_id(self):
         self.info.header_only()
 
     def source(self):
-        tools.get(**self.conan_data["sources"][self.version])
-        extracted_dir = "Lyra-" + \
-            os.path.basename(self.conan_data["sources"][self.version]['url']).replace(
-                ".tar.gz", "")
-        os.rename(extracted_dir, self._source_subfolder)
+        tools.get(
+            **self.conan_data["sources"][self.version],
+            strip_root=True, destination=self._source_subfolder)
 
     def package(self):
         self.copy("LICENSE.txt", dst="licenses", src=self._source_subfolder)
-        self.copy("*.h*", dst="include", src=os.path.join(self._source_subfolder, "include"))
+        self.copy("*.h*", dst="include", src=self._source_subfolder+"/include")
```

Comment:
The use of `os.path.join` was fine, imho.
```suggestion
        self.copy("*.h*", dst="include", src=os.path.join(self._source_subfolder, "include"))
```

Reply:
what is the point of getting rid of import os ? Is it something which should be done as part of regular recipe maintenance ?

Reply:
_insert generic meme about being paid by line of code writen_